dashboard/src/actions/repos.ts

```diff
@@ -119,33 +119,45 @@ export const installRepo = (
   name: string,
   repoURL: string,
   authHeader: string,
+  customCA: string,
 ): ThunkAction<Promise<boolean>, IStoreState, null, AppReposAction> => {
   return async (dispatch, getState) => {
     try {
       const {
         config: { namespace },
       } = getState();
-      let auth;
+      const auth: { header?: any; customCA?: any } = {};
```

Comment:
why any and not string?

Reply:
It's not the text, is the `secretKeyRef` (I can avoid the `any` though)